apps/web/components/apps/AdditionalCalendarSelector.tsx

```diff
@@ -80,11 +78,16 @@ const AdditionalCalendarSelector = ({ isLoading }: AdditionalCalendarSelectorPro
                 width: "max-content",
                 right: "0",
               }),
+              indicatorSeparator: (defaultStyles) => ({
+                ...defaultStyles,
+                display: "none",
+              }),
```

Comment:
We can actually just no render this component by passing in null component  into react select

```
 components={{
        IndicatorSeparator: () => null,
      }}
```

Reply:
NIT and not needed just as a heads up :)